package.json

```diff
@@ -43,12 +43,12 @@
     "test:prune-snapshots": "karma start --prune-snapshots",
     "test:bs": "karma start karma.bs.config.js --coverage",
     "lint": "run-p lint:*",
-    "lint:eclint": "eclint check $(find . \\( -name '*.html' -o -name '*.js' -o -name '*.css' \\) -type f -not -path '*/\\.*' -not -path '*node_modules/*' -not -path '*assets/*' -not -path '*coverage/*')",
+    "lint:eclint": "git ls-files | xargs eclint check",
```

Comment:
it does not cover `-not -path '*assets/*' -not -path '*coverage/*'`, and those files we don't want to lint

Reply:
Ahhh true, I will look into that

Reply:
Alright, I changed `git ls-files` to exclude the assets folder in the result, so now it does not get checked by eclint.

Reply:
did you consider this option? https://github.com/editorconfig/editorconfig/issues/228#issuecomment-163535825

this is a recommened way to exclude folders for eclint by its community https://github.com/jedmao/eclint/issues/80

and we even used this approach in the past internally, for assets as well, before @daKmoR changed it to a shell command alternative

not a big deal for me if it works, just want to make sure this is understandable `git ls-files ':!:assets' | xargs eclint check` and we know how to easily add extra excluded patterns?

Reply:
To exclude more patterns you would just append more ':!:<glob-pattern>' statements and that should work. `git ls-files ':!:assets' ':!:foo'`

I don't mind changing it to excluding folders by placing an editorconfig in the folder but my personal preference is having it more centralized by an exclude pattern in a single config line.
Multiple dummy .editorconfig files with a single line in those folders we need to exclude, it intuitively feels more like a hacky workaround, I'm surprised it's encouraged.

Reply:
Never mind, ':!:assets' like that breaks on Windows. Dummy editorconfig it is 🎉